services/web/server/tests/unit/with_dbs/02/test_projects_ports_handlers.py

```diff
@@ -0,0 +1,292 @@
+            "key": "simcore/services/frontend/parameter/integer",
+            "version": "1.0.0",
+            "label": "Z",
+            "inputs": {},
+            "inputsUnits": {},
+            "inputNodes": [],
+            "parent": None,
+            "thumbnail": "",
+            "outputs": {"out_1": 1},
+            "runHash": None,
+        },
+        "7bf0741f-bae4-410b-b662-fc34b47c27c9": {
+            "key": "simcore/services/frontend/parameter/boolean",
+            "version": "1.0.0",
+            "label": "on",
+            "inputs": {},
+            "inputsUnits": {},
+            "inputNodes": [],
+            "parent": None,
+            "thumbnail": "",
+            "outputs": {"out_1": False},
+            "runHash": None,
+        },
+        "09fd512e-0768-44ca-81fa-0cecab74ec1a": {
+            "key": "simcore/services/frontend/iterator-consumer/probe/integer",
+            "version": "1.0.0",
+            "label": "Random sleep interval_2",
+            "inputs": {
+                "in_1": {
+                    "nodeUuid": "13220a1d-a569-49de-b375-904301af9295",
+                    "output": "output_2",
+                }
+            },
+            "inputsUnits": {},
+            "inputNodes": ["13220a1d-a569-49de-b375-904301af9295"],
+            "parent": None,
+            "thumbnail": "",
+        },
+        "76f607b4-8761-4f96-824d-cab670bc45f5": {
+            "key": "simcore/services/frontend/iterator-consumer/probe/integer",
+            "version": "1.0.0",
+            "label": "Random sleep interval",
+            "inputs": {
+                "in_1": {
+                    "nodeUuid": "08d15a6c-ae7b-4ea1-938e-4ce81a360ffa",
+                    "output": "output_2",
+                }
+            },
+            "inputsUnits": {},
+            "inputNodes": ["08d15a6c-ae7b-4ea1-938e-4ce81a360ffa"],
+            "parent": None,
+            "thumbnail": "",
+        },
+    }
+
+
+@pytest.fixture
+def workbench(workbench_db_column: dict[str, Any]) -> dict[NodeID, Node]:
+    # convert to  model
+    return parse_obj_as(dict[NodeID, Node], workbench_db_column)
+
+
+@pytest.fixture
+def fake_project(
+    fake_project: ProjectDict, workbench_db_column: dict[str, Any]
+) -> ProjectDict:
+    # OVERRIDES user_project
+    project = deepcopy(fake_project)
+    project["workbench"] = workbench_db_column
+    return project
+
+
+@pytest.mark.acceptance_test
+@pytest.mark.parametrize(
+    "user_role,expected",
+    [
+        (UserRole.ANONYMOUS, web.HTTPUnauthorized),
+        (UserRole.GUEST, web.HTTPOk),
+        (UserRole.USER, web.HTTPOk),
+        (UserRole.TESTER, web.HTTPOk),
+    ],
+)
+async def test_io_workflow(
+    client: TestClient,
+    logged_user: UserInfoDict,
+    user_project: ProjectDict,
+    mock_catalog_service_api_responses: None,
+    expected: type[web.HTTPException],
+):
+    """This tests implements a minimal workflow to support the MVP
```

Comment:
MVP stands for?

Reply:
https://en.wikipedia.org/wiki/Minimum_viable_product